datalake_api/fetcher.py

```diff
@@ -84,7 +84,8 @@ def _get_s3_key(self, file_id):
         try:
             return self.s3_bucket.Object(path).get()
         except BotoClienError as e:
-            if e.response['ResponseMetadata']['HTTPStatusCode'] == 404:
+            if 'ResponseMetadata' in e.response and \
+                    e.response['ResponseMetadata']['HTTPStatusCode'] == 404:
                 msg = 'No file with id {} exists'.format(file_id)
```

Comment:
I did, but don't recall. It was while testing on my mac, outside of a virtualenv. I'll stop doing that and remove this change.